homeassistant/components/meteo_france/sensor.py

```diff
@@ -142,29 +144,13 @@ def device_state_attributes(self):
         """Return the state attributes."""
         return {ATTR_ATTRIBUTION: ATTRIBUTION}
 
-    @property
-    def available(self):
-        """Return if state is available."""
-        return self.coordinator.last_update_success
-
-    @property
-    def should_poll(self) -> bool:
-        """No polling needed."""
-        return False
-
     async def async_update(self):
         """Only used by the generic entity update service."""
         if not self.enabled:
             return
 
         await self.coordinator.async_request_refresh()
```

Comment:
Why this method is not part of the CoordinatorEntity? It seems standard to me.

Reply:
Not sure exactly, I have not seen 
```
        if not self.enabled:
            return
```
in many integrations. @bdraco should we include this in `CoordinatorEntity`?

Reply:
This one has a non-standard guard
```
        if not self.enabled:
            return
```

Reply:
@springstan I think we can

Reply:
Not sure if it really is non-standard:
![Screenshot from 2020-08-30 18-12-52](https://user-images.githubusercontent.com/46536646/91664014-9dde6280-eaec-11ea-8afe-9a055ab381bf.png)

Reply:
The code is to avoid refreshing the coordinator when entity not enabled. I'm not the expert of this part. @Quentame  should it be standard ? If not, why do we need this specifically in meteo_france ?

Reply:
@springstan https://github.com/home-assistant/core/pull/39452

Reply:
Exactly as @oncleben31 said.
Fix and merged in PR #39452